.api-breakage/allowlist-branch-fix-deprecations.txt

```diff
@@ -0,0 +1,18 @@
+API breakage: func DatabaseConfigurationFactory.postgres(configuration:maxConnectionsPerEventLoop:connectionPoolTimeout:encodingContext:decodingContext:sqlLogLevel:) has removed default argument from parameter 3
```

Comment:
Shouldn't this file be passed to the api-breakage CI using the `--breakage-allowlist-path` flag (or i see there is an option for it in the reusable CI)  ?

Reply:
Or at second glance, i think you've made the re-useable CI default to this path, so this should be no issue.

Reply:
The reusable breakage check option looks for a file named for the branch being checked